podium-gateway/src/main/java/nl/thehyve/podium/domain/RequestFile.java

```diff
@@ -0,0 +1,120 @@
+package nl.thehyve.podium.domain;
+
+import lombok.AccessLevel;
+import lombok.Setter;
+import nl.thehyve.podium.common.domain.AbstractAuditingEntity;
+import nl.thehyve.podium.enumeration.RequestFileType;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Entity
+@Table(name = "request_file",
+    indexes = {
+        @Index(name = "request_file_created_date_key", columnList = "created_date"),
+    }
+)
+public class RequestFile extends AbstractAuditingEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "request_seq_gen")
+    @GenericGenerator(
+        name = "request_seq_gen",
+        strategy = "org.hibernate.id.enhanced.SequenceStyleGenerator",
+        parameters = {
+            @Parameter(name = "sequence_name", value = "request_seq"),
```

Comment:
Why reuse the request sequence? Maybe make it `request_file_seq` and create such sequence in Liquibase?